java/client/src/main/java/glide/api/RedisClient.java

```diff
@@ -132,4 +133,10 @@ public CompletableFuture<String[]> time() {
         return commandManager.submitNewCommand(
                 Time, new String[0], response -> castArray(handleArrayResponse(response), String.class));
     }
+
+    @Override
+    public CompletableFuture<String> bgsave(boolean schedule) {
```

Comment:
Is schedule not an optional value? Asking for consistency.

Reply:
optional
I set it only if user gives `true` as argument. Should I split into 2 functions?

Reply:
That's what we have done so far. @acarbonetto What are your opinions?

Reply:
sounds like 2 functions